src/HUDL/HUDL.cpp

```diff
@@ -130,21 +112,12 @@ void HUDL::dataForCorner(HUDL::Corner corner, const char* text) {
     sectionColumn += padding;
 
     // Write the text to the screen under the section header.
-    lcd.writeLargeText(text, sectionPage, sectionColumn, false);
-}
-
-void HUDL::writeError(const char* text) {
-    // Calculate the padding to center the text in the section
-    uint8_t length = strlen("!!! Error !!!") * 8;
-    uint8_t padding = (128 - length) / 2;
-
-    // Write the Error header
-    lcd.writeLargeText("!!! Error !!!", 0, padding, false);
-
-    // Write the error message
-    lcd.writeSmallText(text, 2, 0, true);
+    lcd.writeText(text, sectionPage, sectionColumn, EVT::core::DEV::LCD::FontSize::LARGE, false);
 }
 
+// Ignore the -Wreturn-type warning that occurs here because of the switch statements.
```

Comment:
It is yelling because two of the cases are "missing". They are just combined into one line for formatting. A warning here is thrown because it thinks two cases are missing, but they aren't.